content/docs/command-reference/remote/modify.md

```diff
@@ -271,6 +271,43 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   $ dvc remote modify myremote gdrive_client_secret <client secret>
   ```
 
+- `gdrive_use_service_account` - instructs DVC to auth with a
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account).
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_use_service_account <True|False>
+  ```
+
+- `gdrive_service_account_email` - Google Project's
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account)
+  email address.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_email <service account email>
+  ```
+
+- `gdrive_service_account_p12_file_path` - Google Project's
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account)
+  `.p12` file path.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_p12_file_path <file_path.p12>
+  ```
+
+- `gdrive_service_account_user_email` - user email to be impersonated by
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account).
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_user_email <user account email>
+  ```
+
+- `gdrive_trash_only` - configures DVC to move a remote file to the trash or to
+  delete it permanently. By default, is `False`.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_trash_only <True|False>
+  ```
```

Comment:
I don't understand this. Can you explain please, so I can think of a suggestion? Thanks

Also, the values should probably be lower case? i.e. `false` in the bullet, and `<true|false>` in the example.

Reply:
Idea was to specify possible values of the param - True or False. Value from uppercase is a Python style, but I'm not sure that lowercase will be parsed correctly into boolean.

Reply:
I meant please explain what this option does. I don't get it.

As for the letter case, all the other remote params use lower case. The example can probably just be:

  ```dvc
  $ dvc remote modify myremote gdrive_trash_only true
  ```

Reply:
OK found the explanation in iterative/dvc/issues/3477, also Ivan shared some references. Updated!